### DISCUSSION TITLE
BRIDGE_BookClub 7th Book

### DISCUSSION THREAD

**Discussion by fkdl0048** [BRIDGE-DEV/BRIDGE_BookClub#180] [Show and tell]
# BRIDGE_BookClub 7th Book

7회차에 진행할 서적을 미리 추천 받아서 이후 토론이 끝나고 난 후 다수결로 결정하겠습니다.

항상 책을 해당 회차 진행원들과 회의를 통해 결정하기 때문에 이번 6회차에 참여하신 분들은 평소에 읽고 싶었거나 개인적 성장에 도움이 되는 책을 아래 코멘트로 적어주시면 됩니다.

*추천은 참여인원이 아니더라도 가능합니다.*

책을 선정하는 기준은 따로 없고 그때그때 달라지니 편하게 말씀해 주셔도 됩니다.

*개발 관련 or 게임 개발 서적으로 한정합니다.*

**Comment by rpopic2** [comment]
modern effective c++이요... ㅋㅋ

https://www.aladin.co.kr/m/mproduct.aspx?itemid=66541334
